addons/l10n_nz/data/account_tax_report_data.xml

```diff
@@ -85,29 +94,59 @@
                     <record id="tax_report_box12" model="account.report.line">
                         <field name="name">[BOX 12] Multiply BOX11 by 3 and then divide by 23</field>
                         <field name="code">NZBOX12</field>
+                        <field name="sequence">220</field>
                         <field name="aggregation_formula">(NZBOX11.balance * 3)/23</field>
                     </record>
                     <record id="tax_report_box13" model="account.report.line">
                         <field name="name">[BOX 13] Credit adjustments from your calculation sheet</field>
                         <field name="code">NZBOX13</field>
+                        <field name="sequence">230</field>
                         <field name="expression_ids">
-                            <record id="tax_report_box13_formula" model="account.report.expression">
+                            <record id="tax_report_box13_aggregation" model="account.report.expression">
```

Comment:
Useless change. That's actually what cause the runbot failure.

Reply:
It's because we try to keep the "...box13_formula" id linked to its manual value, so that we don't lose it after the tax report has changed

Reply:
Can you confirm if it is stable friendly or not @KangOl ? Need to know if I can add the exception.

Reply:
Nevermind, no exception could make it pass.